packages/design-system/src/components/Form/Affix/variations/AffixSelect.tsx

```diff
@@ -18,9 +20,9 @@ const AffixSelect = forwardRef((props: AffixSelectPropsType, ref: Ref<HTMLSelect
 			selectProps: Omit<SelectProps, 'hasError' | 'name' | 'children' | 'label'>,
 		) {
 			return (
-				<Select id={fieldID} {...selectProps} isAffix ref={ref}>
+				<SelectNoWrapper id={fieldID} {...selectProps} isAffix isSuffix={isSuffix} ref={ref}>
```

Comment:
This is the new version that does not import InputWrapper, and therefore does not import AffixSelect.

Reply:
♻️ 🚫